src/argus_htmx/models.py

```diff
@@ -1,3 +1,3 @@
-from django.db import models
+from django.db import models  # noqa: F401 - unused-import
```

Comment:
This file will be used by the userprefs-branch and should stay even if it was autogenerated.